ssv/committee.go

```diff
@@ -100,17 +101,36 @@ func (c *Committee) ProcessMessage(signedSSVMessage *types.SignedSSVMessage) err
 		if err := qbftMsg.Decode(msg.GetData()); err != nil {
 			return errors.Wrap(err, "could not get consensus Message from network Message")
 		}
-		runner := c.Runners[spec.Slot(qbftMsg.Height)]
-		// TODO: check if runner is nil
+
+		if err := qbftMsg.Validate(); err != nil {
+			return errors.Wrap(err, "invalid qbft Message")
+		}
+
+		runner, exists := c.Runners[spec.Slot(qbftMsg.Height)]
+		if !exists {
+			return errors.New("no runner found for message's slot")
+		}
```

Comment:
This is ok...
But maybe we should add a comment that this could also be a future message.
We need to have this comment for the TL reviewers at the very least on the final review